src/styles/blocks/form.scss

```diff
@@ -0,0 +1,60 @@
+.form {
+  display: flex;
+  flex-direction: column;
+
+  &__label {
+    width: 100%;
+  }
+
+  &__input {
+    color: $color-white;
+    font-size: 15px;
+    font-family: $Poppins-font-familly;
+    font-style: normal;
+    font-weight: 600;
+    line-height: normal;
+    background-color: $color-dark;
+    border: 0;
+    border-bottom: 2px solid $color-white;
+    padding: 0;
+    outline: none;
+    padding-bottom: 16px;
+    width: 100%;
+    margin-bottom: 40px;
+    @include hover (border-color, $color-blue);
+
+    @include onTablet {
+      margin-bottom: 48px;
+    }
+
+    &--size {
+      resize: none;
+      padding: 0;
+      height: 41px;
+      margin-bottom: 0;
+    }
+  }
+
+  &__button {
```

Comment:
You are repating this button styles in each section (vision, strategic, form), when there will be a need to modify the button you will need to make changes in 3 places. You could extract button styles to **button** block and have it in one place. Try to do it so all buttons behaves the same